CypressTests/cypress/e2e/Get-UpdateApplication.cy.js

```diff
@@ -0,0 +1,269 @@
+/// <reference types="Cypress" />
+import {UnauthorisedUserCannotUpdatePayload} from '../support/payloads/UnauthorisedUserCannotUpdateBody.spec'
```

Comment:
One for a future enhancement - as these payloads are very similar but with a few changed bits of data, could we instead have one import of a positive test payload and just modify the data within the test?

e.g.
import { goodPayload } from '/path/to/payload'
[...]
let somePayload = goodPayload         // uses the happy path payload
somePayload.someField = 'badData'  // now modify data for failure condition